src/CalculateUtil.js

```diff
@@ -0,0 +1,28 @@
+import BASIC_SEPARATOR from "./constants/basicSeparator";
+
+const CalculateUtil = {
+  parseInput(input) {
+    const separator = [...BASIC_SEPARATOR];
+    const re = /(\/\/).+(\\n)/;
```

Comment:
```suggestion
    const customSeparatorRegExp = /(\/\/).+(\\n)/;
```
`re`라는 변수명이 어떤 것을 의미하는지 파악하기 어려운 것 같아서 커스텀 구분자 정규 표현식이라는걸 알리는건 어떨까요??

Reply:
그렇네요.. 훨씬 좋은 것 같습니다!! 역시 네이밍은 아직 갈길이 먼 것 같네요ㅠ 그래도 우혁님 덕분에 정말 많이 늘고 있어요! 감사해요 ㅎㅎ 👍